site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
-### Review violation troubleshooting updates
-
-#### Published 2021.10.27
+### Chrome 96: Dynamic content scripts {: #m96-dynamic-content-scripts }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting/) API now supports
+[registering](/docs/extensions/reference/scripting/#method-registerContentScripts),
+[updating](/docs/extensions/reference/scripting/#method-updateContentScripts),
+[unregistering](/docs/extensions/reference/scripting/#method-unregisterContentScripts), and [getting
+a list of content scripts](/docs/extensions/reference/scripting/#method-getRegisteredContentScripts)
+at runtime. Previously, content scripts could only be statically declared in an extension's
+manifest.json or programmatically injected at runtime with
+[`chrome.scripting.executeScript`](/docs/extensions/reference/scripting/#method-executeScript).
+
+### Chrome 96: Expanded promise support {: #m96-promise-support }
```

Comment:
We increased support here by some non-trivial factor.  Let's add some kind of qualifier - "Significantly expanded" for less fun, "Wildly expanded" (or similar) for more fun.

Reply:
Happy to! I think in my original draft the intro for this section I said something like "This release contains more promise updates than any previous release." I think I'll change this header and tweak the intro paragraph to lean more into this.